### PR TITLE
Pass baggage in ZCF through buildRootObject to start()

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
refs: #5593
refs: #5635

## Description

Pass the `baggage` provided to `buildRootObject()` to `startContract()`, which calls `start()`, to make it available to the zoe contract.

### Security Considerations

contract has its own sub-baggage.

### Documentation Considerations

When we write up durability, the Zoe contract interface needs to mention that the baggage comes in a separate argument after `privateArgs`.

### Testing Considerations

We'll need a test showing upgrade of a contract, but we already knew that.

### CODE REVIEW

File: packages/zoe/src/contractFacet/vatRoot.js [hunk 1]
```diff
@@ -44,11 +48,15 @@ export function buildRootObject(powers) {
       testJigSetter,
     );
     zcfZygote.evaluateContract(bundleOrBundleCap);
+    const contractBaggage = provide(baggage, 'contractBaggage', () =>
+      makeScalarBigMapStore('zoeContract'),
+    );
```
Comment (erights) [inline packages/zoe/src/contractFacet/vatRoot.js:53]:
```suggestion
    const contractBaggage = provideDurableMapStore(baggage, 'contractBaggage');
```

File: packages/zoe/src/contractFacet/internal-types.js [hunk 2]
```diff
@@ -11,5 +11,6 @@
  *     instanceRecordFromZoe: InstanceRecord,
  *     issuerStorageFromZoe: IssuerRecords,
  *     privateArgs?: object,
+ *     baggage?: MapStore<string,any>,
```
Comment (erights) [inline packages/zoe/src/contractFacet/internal-types.js:14]: Also the `startContract` implementation in zcfZygote, right?

**Review by erights — APPROVED** [review]
LGTM

**Comment by Chris-Hibbert** [comment]
#5635 will include what's necessary.

**Pull request closed without merging**